src/cs2_sdk/entity/ccsweaponbase.h

```diff
@@ -21,6 +21,27 @@
 
 #include "cbaseentity.h"
 
+enum gear_slot_t : uint32_t
+{
+	GEAR_SLOT_INVALID = 0xffffffff,
```

Comment:
Can I have eyes on this? This is actually `GEAR_SLOT_INVALID = 0xffffffffffffffff` in the [schema dump](https://github.com/neverlosecc/source2sdk/blob/b98f5071a6246d813dd5d0d5cb68e7c2dc62a811/sdk/server.hpp#L1842), however the original value didn't actually fit in `uint32_t` and failed builds, so I shortened it.

Reply:
I'd say this is an issue with the dumper, that value should be fine.